formal-spec/measurements.tex

```diff
@@ -141,8 +143,9 @@ \section{Impact on performance}
       ylabel={Usage percentage},
       ymin=0.0,
       ymax=250.0,
-      ymode=linear,
-      ytick={5.0, 20.0, 50.0, 100.0, 150.0, 200.0, 250.0},
+      ymode=log, %linear,
```

Comment:
Why the change in the scale?

Also remove the code that isn't needed.

Reply:
I think that the log scale depicts better that an increase by 10 times in the number of participants , increases by 10x the usage percentage. Log scales show relative values (increase by 10x) instead of absolute ones and (at least to me) it is more intuitive.